src/components/Layout/Header/NavBar/Notifications.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, IconButton, useColorMode } from '@chakra-ui/react'
+import type { CustomTheme } from '@wherever/react-notification-feed'
+import {
+  NotificationBell,
+  NotificationFeed,
+  NotificationFeedProvider,
+} from '@wherever/react-notification-feed'
+import { getConfig } from 'config'
+import { useMemo } from 'react'
+import { KeyManager } from 'context/WalletProvider/KeyManager'
+import { getLocalWalletType } from 'context/WalletProvider/local-wallet'
+import { useFeatureFlag } from 'hooks/useFeatureFlag/useFeatureFlag'
+import { breakpoints, theme } from 'theme/theme'
+
+const WHEREVER_ENABLED_WALLETS: KeyManager[] = [
+  KeyManager.TallyHo,
```

Comment:
currently only enabled for EIP-1193 wallets

https://eips.ethereum.org/EIPS/eip-1193

at some point, probably soon, we're going to have to add this concept to hdwallet for native and keepkey wallets so we're more metamask-like

Reply:
Got this one for ya if we feel like writing a type guard https://github.com/shapeshift/web/blob/4e8c9ca946e3d9cef138a714efb029f868aa8b84/src/context/WalletProvider/WalletProvider.tsx#L90-L95